lib/Subscription/CaptchaRenderer.php

```diff
@@ -84,11 +101,20 @@ public function getCaptchaPageContent() {
     // subscription form
     $form_html .= FormRenderer::renderBlocks($form, $honeypot = false);
     $form_html .= '</div>';
-    $form_html .= '<div class="mailpoet_message">';
-    $form_html .= '<p class="mailpoet_validate_success" style="display:none;">' . $form_model['settings']['success_message'] . '</p>';
-    $form_html .= '<p class="mailpoet_validate_error" style="display:none;"></p>';
-    $form_html .= '</div>';
+    $form_html .= $this->renderFormMessages($form_model, $show_success_message, $show_error_message);
     $form_html .= '</form>';
     return $form_html;
   }
+
+  private function renderFormMessages(
```

Comment:
This method and its usage are a bit strange:
- Two boolean arguments deciding what to render while we actually never render both of the messages.
- Hiding message by `display: none` instead not rendering it at all.

Since Alex is out I'm taking over to fix this and assign to someone else for review.